config/routes/bookings.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file contains the configuration for the routes related to bookings in the EDD Bookings.
+ *
+ * @since [*next-version*]
+ */
+
+/*
+ * The route for querying bookings.
+ *
+ * @since [*next-version*]
+ */
+$cfg['eddbk_rest_api']['routes'][] = [
+    'pattern' => '/bookings',
```

Comment:
Can you explain how `^` and `$` (end and start of string) are interpreted here, can they be included, and why?

Reply:
This regex is appended to another, larger regex by WordPress. So the start and end of string characters cannot be used here. The whole URL of the request is matched to the larger regex.

Also FYI, WordPress uses the `!` delimiter for regex:

https://core.trac.wordpress.org/browser/tags/4.9/src/wp-includes/rest-api/class-wp-rest-server.php#L516

Reply:
_Outdated_